src/main/resources/message/messages_de_DE.properties

```diff
@@ -328,6 +330,7 @@ configuration.validate.double=Der Wert f\u00fcr das Feld {field} ist keine Dezim
 configuration.validate.integer=Der Wert f\u00fcr das Feld {field} muss zwischen 0 und 2E31-1 liegen.
 configuration.validate.localPath=MoPat besitzt f\u00fcr das Feld {field} keine Lese-/Schreibrechte.
 configuration.validate.long=Der Wert f\u00fcr das Feld {field} muss zwischen 0 und 2E63-1 liegen.
+configuration.validate.mappedConfigurationNotFound=Der Wert f�r diese Konfiguration wurde nicht ausgew�hlt.
```

Comment:
There seem to be some encoding problems here. IntelliJ likes to destroy our Umlauts, thats why we encode them like so \u00fc